src/net/sourceforge/kolmafia/session/GreyYouManager.java

```diff
@@ -486,7 +487,7 @@ public static class GooSkill extends Absorption {
     private final int level;
 
     private String enchantments = "";
-    private String modsLookup = "";
+    private Modifiers.Lookup modsLookup = new Modifiers.Lookup(ModifierType.GENERATED, "");
```

Comment:
(nothing to address here; just noting that you're assigning the modifier type GENERATED unilaterally here and in a few other cases.)

Reply:
Yep, this should be type NONE I think.